interface/i18n/mist.it.i18n.json

```diff
@@ -48,7 +48,7 @@
                 "mainNetwork": "Rete principale",
                 "startMining": "⛏ Inizia mining",
                 "stopMining": "⛏ Ferma mining",
-                "openRemix": "Open Remix IDE"
+                "openRemix": "Apri IDE Remix",
```

Comment:
The travis build is failing when loading this json file because of this trailing comma

Reply:
If you can remove the trailing comma in a commit the build should pass and we can merge in :)